pkg/syncmap/syncmap.go

```diff
@@ -105,6 +105,20 @@ func (mmap *SyncMap) GetAsInt(key string) (int, error) {
 	return 0, errors.Wrapf(errortypes.ErrConvertFailed, "key %s: %v", key, v)
 }
 
+// GetAsInt64 returns result as int64.
+// The ErrConvertFailed error will be returned if the assertion fails.
+func (mmap *SyncMap) GetAsInt64(key string) (int64, error) {
+	v, err := mmap.Get(key)
+	if err != nil {
+		return 0, errors.Wrapf(err, "key: %s", key)
```

Comment:
should make error message more readable.
You see your error message is "key: a: blablabla".
I think much better way is "failed to get key a in map: blablabla". So `errors.Wrapf(err, "failed to get key %v in map", key)`?

Reply:
Done. 😄